JS/removeItem.js

```diff
@@ -0,0 +1,33 @@
+
+function removeInCart(arr, id) {
+    const myArr = [...arr];
+    let idx = myArr.findIndex((e) => {
+        if (e.id == id)
+            return true;
+    });
+    if (idx == -1) return;
+    myArr.splice(idx, 1);
+    return myArr;
+}
+
+const DeleteProduct = (data, id) => {
```

Comment:
`removeInCart` and `DeleteProduct` are doing the same exact thing, so it can be merged into one function that takes a list of items and and id, and return a new array with the item of that id deleted, then we can use this function to remove items in the seller page or the cart page.